scripts/skills/actives/rf_pocket_sand_skill.nut

```diff
@@ -0,0 +1,82 @@
+this.rf_pocket_sand_skill <- this.inherit("scripts/skills/actives/throw_dirt_skill", {
+	m = {
+		// Private
+		RemainingUses = 0
+	},
+	function create()
+	{
+		this.throw_dirt_skill.create();
+
+		this.m.ID = "actives.rf_pocket_sand_skill";
+		this.m.Name = "Throw Pocket Sand";
+	}
+
+	function getTooltip()
+	{
+		local ret = this.throw_dirt_skill.getTooltip();
+
+		if (this.getContainer().getActor().isPlacedOnMap())
+		{
+			if (this.getRemainingUses() == 0)
+			{
+				ret.push({
+					id = 20,
+					type = "text",
+					icon = "ui/tooltips/warning.png",
+					text = "Remaining uses: " + ::MSU.Text.colorRed("0"),
+				});
+			}
+			else
+			{
+				ret.push({
+					id = 20,
+					type = "text",
+					icon = "ui/icons/special.png",
+					text = "Remaining uses: " + ::MSU.Text.colorGreen(this.getRemainingUses()),
+				});
+			}
+		}
+		else
+		{
+			ret.push({
+				id = 21,
+				type = "text",
+				icon = "ui/icons/special.png",
+				text = "You have one use for each empty bag slot that you start the battle with",
+			});
+		}
+
+		return ret;
+	}
+
+	function isUsable()
+	{
+		return this.throw_dirt_skill.isUsable() && (this.m.RemainingUses != 0);
```

Comment:
Please remove the redundant parantheses

Reply:
I use these redundant parantheses to improve readability. 
Please provide me with a hard rule about this so I know when I'm allowed to put optional parantheses